react/Text/Text.sketch.js

```diff
@@ -3,26 +3,26 @@ import mapKeys from 'lodash/mapKeys';
 import Text from './Text';
 
 export const text = {
-  'Hero': <Text hero>Hero text</Text>,
-  'Headline': <Text headline>Headline text</Text>,
-  'Heading': <Text heading>Heading text</Text>,
-  'Subheading': <Text subheading>Subheading text</Text>,
-  'Standard': <Text>Standard text</Text>,
-  'Standard Positive': <Text positive>Standard positive text</Text>,
-  'Standard Critical': <Text critical>Standard critical text</Text>,
-  'Standard Secondary': <Text secondary>Standard critical text</Text>,
-  'Standard Strong': <Text strong>Standard strong text</Text>,
-  'Superstandard': <Text superstandard>Superstandard text</Text>,
-  'Superstandard Positive': <Text superstandard positive>Superstandard positive text</Text>,
-  'Superstandard Critical': <Text superstandard critical>Superstandard critical text</Text>,
-  'Superstandard Secondary': <Text superstandard secondary>Superstandard critical text</Text>,
-  'Superstandard Strong': <Text superstandard strong>Superstandard strong text</Text>,
-  'Substandard': <Text substandard>Substandard text</Text>,
-  'Substandard Positive': <Text substandard positive>Substandard positive text</Text>,
-  'Substandard Critical': <Text substandard critical>Substandard critical text</Text>,
-  'Substandard Secondary': <Text substandard secondary>Substandard critical text</Text>,
-  'Substandard Strong': <Text substandard strong>Substandard strong text</Text>,
-  'Small': <Text small>Small text</Text>
+  '1. Hero — Desktop: 42px, Mobile: 21px': <Text hero>Hero text</Text>,
```

Comment:
These values smell like something that could rot, what about importing the less vars and multiplying them by the base font size so these update automatically?

Reply:
Yeah, I had the same concern, but was worried that the solution would be too much of a headache. I'll have a look.

(Wish we had these available in JS rather than Less)

Reply:
Done! Nice suggestion, feel so much better about this now 👍